src/objects/spawn_point.gd

```diff
@@ -0,0 +1,26 @@
+extends Node3D
+class_name SpawnPoint
+
+
+@onready var exclusion_zone = $ExclusionZone
+
+
+# The player currently associated with this spawn point, or -1 if there are no players.
+var player_id: int = -1
+
+
+func _ready():
+	hide()
+
+
+# Returns true if there are no players (except possibly the player that is spawning) within the
+# exclusion zone.
+# :param player_id: The player that is spawning.
+func available(_player_id: int) -> bool:
```

Comment:
I realize it's to avoid the name collision with the variable `player_id`, but usually by convention, prefixing a variable name with an `_` by convention means that variable is being unused, and it's being used here. I also kinda don't like that the `player_id` variable is never set/read from within this class, but only from Game which feels a little weird

Reply:
Fixed